### PR TITLE
RECIPE-85

### PR DESCRIPTION
**Pull request by PatsiChang**
Added Bootstrap and wrapper divs
Link: https://smart-home-cloud.atlassian.net/browse/RECIPE-85

### CODE REVIEW

File: new/components/basic/layout/BaseBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useStyle } from "@/hooks/navigation/useTheme";
+import { PropsWithChildren } from "react";
+import { View, ViewStyle } from "react-native";
+
+interface BaseBlockProps extends PropsWithChildren<{}> {
+    style?: ViewStyle,
+}
+
+const BaseBlock = ({ children, style, ...props }: BaseBlockProps) => {
+    const styleObj = useStyle("baseBlock");
```
Comment (ChakShingLo) [inline new/components/basic/layout/BaseBlock.tsx]: look like styleObj is never been used

File: new/components/basic/layout/BaseHeader.tsx [hunk 2]
```diff
@@ -1,19 +1,12 @@
...
+import { BaseLargeText } from "./BaseText";
 
 
 interface BaseHeaderProps extends PropsWithChildren<{}> {
 }
 
 export default function BaseHeader({ children, ...props }: BaseHeaderProps) {
     return (
-        <Text style={styles.header}>{children}</Text>
+        <BaseLargeText>{children}</BaseLargeText>
```
Comment (ChakShingLo) [inline new/components/basic/layout/BaseHeader.tsx]: what is the purpose for wrapping BaseLargeText with BaseHeader?
If there is no extra logic added to it

File: new/hooks/navigation/useTheme.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { darkTheme, lightTheme } from "@/app/stylesheet";
+import { ThemeContext } from "@/context/ThemeContext";
+import { useContext } from "react";
+
+export const useTheme = () => useContext(ThemeContext);
+
+export const useStyle = (name: string): any => {
```
Comment (ChakShingLo) [inline new/hooks/navigation/useTheme.tsx]: why this file is placed under `/hooks/navigation` ?

File: new/components/basic/layout/BaseBlock.tsx (hunk 1 above)
Comment (PatsiChang) [reply new/components/basic/layout/BaseBlock.tsx]: i removed it now :D

**Review by ChakShingLo — APPROVED** [review]
lgtm

**Pull request merged**